### DISCUSSION TITLE
How i can use pretrained model for imagenet-200 detection dataset?

### DISCUSSION THREAD

**Discussion by afzaalhussain50** [facebookresearch/detectron2#3372] [Q&A]
## 🚀 Feature
i want to use pretrained faster-rcnn model for imagenet-200 object detection dataset with 200 classes, can you guide how i can do that?
Is there any pretrained model available for imagenet-200 dataset?

Because i want to use pretrained faster-rcnn model for COCO-80 classes and imagent-200 classes datasets for research work.

Regards,
Afzaal

**Comment by akiross** [comment]
Hi, did you find a solution? I couldn't find an example or explanation on how to do so, but I believe it is sufficient to specify the path of the weights in the configuration, e.g.

```
    cfg.MODEL.WEIGHTS = "https://dl.fbaipublicfiles.com/detectron2/ImageNetPretrained/MSRA/R-101.pkl"
```

instead of getting them from the model zoo configuration. This worked for me (meaning: the training did not crash). In the `MODEL_ZOO.md` file you can find [links to pretrained imagenet weights](https://github.com/facebookresearch/detectron2/blob/main/MODEL_ZOO.md#imagenet-pretrained-models).

I'm still learning about detectron2 so I'm not sure this is all fine and dandy, but I hope it can help to get started.
